### DISCUSSION TITLE
Tracking in RVIZ

### DISCUSSION THREAD

**Discussion by GideonLombardo** [USC-ACTLab/crazyswarm#632] [Q&A]
Hello, I have recently gotten my Crazyswarm setup to work. 

However, I notice that the axes for my CF are not stable in RVIZ. I don't know why the axes are unstable, because the Vicon System shows the CF being completely stable. Does anyone have guidance on this issue? Please let me know.

**Comment by whoenig** [comment]
Which motion tracking mode are you using?

**Reply by GideonLombardo** [reply]
Sorry for the delayed response. Finals have just ended at my university and I was able to get back into the lab. I am using Vicon to track my CFs, and we use unique marker configurations in our lab. Therefore, I am using "motionCapture" as my object tracking type.
